src/molecules/Amount/Amount.native.js

```diff
@@ -122,15 +156,23 @@ const Amount = ({ size, testID, children, currency }) => {
 
 Amount.propTypes = {
   children: PropTypes.string.isRequired,
-  size: PropTypes.oneOf(['medium', 'large', 'xlarge', 'xxlarge', 'xxxlarge']),
+  size: PropTypes.oneOf(['xsmall', 'medium', 'large', 'xlarge', 'xxlarge', 'xxxlarge']),
   testID: PropTypes.string,
   currency: PropTypes.oneOf(geISOCurrencyList()),
+  variant: PropTypes.oneOf(['camel', 'normal']),
+  weight: PropTypes.oneOf(Object.keys(baseTheme.fonts.weight)),
+  variantColor: PropTypes.oneOf(getVariantColorKeys()),
+  subtle: PropTypes.bool,
 };
 
 Amount.defaultProps = {
   size: 'medium',
   testID: 'ds-amount',
   currency: 'INR',
+  variant: 'camel',
+  weight: 'bold',
+  variantColor: 'shade',
+  subtle: false,
```

Comment:
```suggestion
  weight: 'bold',
  variant: 'camel',
  variantColor: 'shade',
  subtle: false,
```

just better code readability when someone directly looks at the API. same applies to propTypes